If something we should mark it as broken on darwin due to the build failure there:
```
checking whether to enable inotify support... yes, shared
configure: error: Your system does not support inotify
builder for '/nix/store/20hjnm3ajb6svaq09dmh4a96ln8x7kml-php-inotify-3.0.0.drv' failed with exit code 1
```

--- a/pkgs/development/php-packages/inotify/default.nix
+++ b/pkgs/development/php-packages/inotify/default.nix
@@ -1,0 +1,21 @@
+{ buildPecl
+, lib
+, stdenv
+}:
+
+buildPecl {
+  pname = "inotify";
+
+  version = "3.0.0";
+  sha256 = "sha256-xxt4ZEwBFVecx5T1jnhEFEF1HXgEC52dGiI9Ppwtcj0=";
+
+  doCheck = true;
+
+  meta = with lib; {
+    broken = stdenv.isDarwin; # no inotify support
+    description = "Inotify bindings for PHP";
+    license = licenses.php301;
+    homepage = "https://github.com/arnaud-lb/php-inotify";
+    maintainers = teams.php.members;
+  };
+}

--- a/pkgs/top-level/php-packages.nix
+++ b/pkgs/top-level/php-packages.nix
@@ -216,6 +216,8 @@ lib.makeScope pkgs.newScope (self: with self; {
 
     imagick = callPackage ../development/php-packages/imagick { };
 
+    inotify = callPackage ../development/php-packages/inotify { };
+
     mailparse = callPackage ../development/php-packages/mailparse { };
 
     maxminddb = callPackage ../development/php-packages/maxminddb { };

